base/v0_1/translate_test.go

```diff
@@ -0,0 +1,278 @@
+// Copyright 2019 Red Hat, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.)
+
+package v0_1
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/coreos/ignition/v2/config/util"
+	"github.com/coreos/ignition/v2/config/v3_0/types"
+)
+
+// Most of this is covered by the Ignition translator generic tests, so just test the custom bits
+
+func TestTranslateFile(t *testing.T) {
```

Comment:
nit: I'm a fan of giving tests a simple document string to make it a bit easier to see at a glance what is meant to be tested.